pallets/aleph/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+//! Aleph session manager.
+//!
+//! This pallet manages the changes in the committee responsible for establishing consensus.
+//! Currently, it's PoA where the validators are set by the root account. In the future, a new
+//! pallet for PoS elections will replace this one.
+//!
+//! For full integration with Aleph finality gadget, the `primitives::AlephSessionApi` should be implemented.
```

Comment:
This whole comment is at least partially outdated.

Reply:
Heavily! I'll change this and pallet elections doc.

Reply:
Changed.